example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Thu Dec 10 14:52:09 CET 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```

Comment:
can we make sure this make the library usable for RN lower than 0.64?

Reply:
Well I've tried to set it to 4.2, but I'm getting the following error (in the `example/` app):
![image](https://user-images.githubusercontent.com/15199031/101795403-261acd80-3b08-11eb-8392-43aad9710e5d.png)

Reply:
Also, according to the [rn upgrade helper](https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.0-rc.1), RN 0.63 uses `gradle-6.2-all.zip` so I'd say 6.1.1 is fine

Reply:
mhm, I see that 6.2 is used on RN 0.63, so it's not too bad. Would be cool to test on lower RN versions (but up to 0.61 max). Will require a major bump anyway

Reply:
0.61 uses gradle 5, 0.62 uses gradle 6 - I don't have a lot of android experience, but shouldn't it build either way?